packages/shared/src/components/modals/ShareModal.tsx

```diff
@@ -96,7 +101,7 @@ export default function ShareModal({
         />
         <p className="py-2.5 font-bold typo-callout">Share via</p>
         <SocialShare
-          post={post}
+          post={article}
```

Comment:
I think this one should be reverted as well

Reply:
The link will be restricted to most people outside these social platforms. They might think the link is broken as they are not part of the Squad that can view the post.